com.unity.multiplayer.mlapi/Runtime/AOI/ClientObjectMap.cs

```diff
@@ -0,0 +1,124 @@
+// Although this is (currently) inside the MLAPI package, it is intentionally
+//  totally decoupled from MLAPI with the intention of allowing it to live
```

Comment:
Can you expand on why you're going this route? What is the proposed use for a package that does AOI but doesn't depend on MLAPI types? A separate package makes sense but I'm missing the point of no dependencies here. Would also love to understand how you see the full decoupling happening given that it's a hard dependency of MLAPI based on integration within NetworkedBehavior.cs